api/AbsoluteOrientationSensor.json

```diff
@@ -5,34 +5,40 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/AbsoluteOrientationSensor",
         "support": {
           "chrome": {
-            "version_added": "69"
+            "version_added": "67"
           },
           "chrome_android": {
-            "version_added": "69"
+            "version_added": "67"
           },
           "edge": {
-            "version_added": "≤79"
+            "version_added": "79"
           },
           "firefox": {
             "version_added": false
           },
+          "firefox_android": {
+            "version_added": false
+          },
           "ie": {
             "version_added": false
           },
           "opera": {
-            "version_added": "56"
+            "version_added": "54"
           },
           "opera_android": {
             "version_added": "48"
```

Comment:
As noted in https://github.com/mdn/browser-compat-data/pull/6695, opera_android here is correct even though it can *look* like an oversight in the change.